packages/mollie-integration/src/mollie-script-loader.ts

```diff
@@ -0,0 +1,27 @@
+import { ScriptLoader } from '@bigcommerce/script-loader';
+
+import { PaymentMethodClientUnavailableError } from '@bigcommerce/checkout-sdk/payment-integration-api';
+
+import { MollieClient, MollieHostWindow } from './mollie';
+
+export default class MollieScriptLoader {
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    constructor(private scriptLoader: ScriptLoader, private _window: Window = window) {}
+
+    isMollieWindow(window: Window): window is MollieHostWindow {
+        return 'Mollie' in window;
+    }
```

Comment:
I think it is wrong place for this typeguard method, so we might move it to a separate file (`mollie-utils.ts` or `is-mollie-window.ts` for example)

Reply:
Done